infrastructure/docker-registry/README.md

```diff
@@ -1,74 +1,75 @@
 
-## Docker Registry Helm Chart
-We used the [Docker Registry Helm Chart](https://github.com/helm/charts/tree/master/stable/docker-registry), that is a Kubernetes chart to deploy a private Docker Registry where we have appropriately modified the values in [values.yaml](https://github.com/helm/charts/blob/master/stable/docker-registry/values.yaml) file.
+## Harbor Registry Helm Chart
+To install Harbor, it is possible to leverage the [official Helm Chart](https://github.com/helm/charts/tree/master/stable/docker-registry), appropriately configuring the `values.yaml` file (additional details follow in the next sections).
 
-### Configuration
-These are our modification of the various file.
+### Pre-requisites
 
-  1. Annotations concerning the authentication in the Ingress controller.
-```yaml
-annotations:
-   nginx.ingress.kubernetes.io/auth-realm: Authentication Required - ok
-   nginx.ingress.kubernetes.io/auth-secret: basic-auth
-   nginx.ingress.kubernetes.io/auth-type: basic
-```
+  1. [Kubernetes cluster 1.10+](https://kubernetes.io/releases/download/)
+  2. [Helm 3](https://helm.sh/docs/intro/install/)
+  3. [High available ingress controller](https://github.com/netgroup-polito/CrownLabs/tree/master/infrastructure/ingress-controller) (Harbor does not manage the external endpoint)
+  4. [High available PostgreSQL 9.6+](#https://github.com/netgroup-polito/CrownLabs/tree/master/infrastructure/identity-provider#postgresql-operator) (Harbor does not handle the deployment of HA of database)
+  5. [High available Redis](#https://github.com/bitnami/charts/tree/master/bitnami/redis/#installing-the-chart) (Harbor does not handle the deployment of HA of Redis)
+  6. PVC that can be shared across nodes (i.e., with `ReadWriteMany` access mode) or external object storage
 
-  2. Annotations concerning the configuration of the Ingress controller.
-```yaml
-ingress:
-  enabled: true
-  path: /
-  hosts:
-    - <registry.domain.name>
-  annotations:
-    cert-manager.io/cluster-issuer: letsencrypt-production
-  labels: {}
-  tls:
-    - hosts:
-      - <registry.domain.name>
-```
+### Redis Configuration
+For the installation of Redis service, please see the link on the pre-requisites section. In our architecture we have a [Redis-Sentinel](https://redis.io/topics/sentinel) service, instead of [Redis Cluster](https://redis.io/topics/cluster-tutorial) by setting the following parameter in the file values of redis_service_values.yaml file. In the file you have to set an IP for the ClusterIP of the sentinel where after harbor is attached in the cluster and a name for your master. If you want have the replicas of redis running in different nodes you have to set the replica.podAntiAffinityPreset parameter to soft or hard, depending what you need. Please see this [link](https://docs.bitnami.com/tutorials/assign-pod-nodes-helm-affinity-rules/) to see the differences.
```

Comment:
Some comments:
* Please, write each sentence on its own line, so that it is better handled by git
* Which link are you referring to?
* When you say something, add also a motivation (why is it used sentinel instead of redis cluster?)
* Add the full command to install/upgrade redis, so that it is easier to upgrade it in future (`helm upgrade --install ...`)